HW2-task1.py

```diff
@@ -0,0 +1,24 @@
+""" Finding palindrome
+Write a program that check whether a number is palindrome or Not.
+Input string contains only numbers.
+Please work only arithmetic operations, loops and if-condition.
+"""
+
+number = int(input("Enter number: \n"))
+if number:
+    len_of_number = 0
```

Comment:
maybe it would be easier to revert a number with just one for loop? 
this solution looks too complex